src/main/scala/advent/solutions/Day1.scala

```diff
@@ -1,46 +1,55 @@
 package advent.solutions
 
+import scala.annotation.tailrec
+
 /** Day 1: The Tyranny of the Rocket Equation
```

Comment:
I've added a formatter (`scalafmt`) so we'll not need to worry about formatting changes.  Could you fetch the most recent changes and then run `sbt scalafmtAll`?